vllm/executor/multiproc_gpu_executor.py

```diff
@@ -38,7 +38,7 @@ def _init_executor(self) -> None:
             "please set tensor_parallel_size to less than max local gpu count")
 
         distributed_init_method = get_distributed_init_method(
-            get_ip(), get_open_port())
+            "127.0.0.1", get_open_port())
```

Comment:
add a comment explaining why we can hardcode this? `Multi-node not supported by multiproc executor`

Reply:
added in [b949c51](https://github.com/vllm-project/vllm/pull/5619/commits/b949c51be05efcd7bbbc2531277857a56ff43606)